roles/java/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ java_tar_file: >-
   OpenJDK{{ java_major }}U-jdk_{{ java_arch }}_linux_hotspot_{{ java_core }}_{{ java_build }}.tar.gz
 java_url: "{{ java_download_base }}/{{ java_tar_file }}"
 java_checksum: sha256:{{ java_url }}.sha256.txt
-java_home: /opt/openjdk-{{ java_core }}
+java_home: /opt/java
```

Comment:
I think it's quite risky for the idempotency, I think we should keep as it is here, and (optionally?) create a symlink to latest installed version which point to /opt/java, then the other roles can use /opt/java as default value.